soap/soap.go

```diff
@@ -217,6 +228,15 @@ func WithHTTPHeaders(headers map[string]string) Option {
 	}
 }
 
+// WithMTOM is an Option to set Message Transmission Optimization Mechanism
+// When enabled, fields of type Binary will be encoded using XOP.
+// When disabled, those fields are transmitted with Base64 encoding.
+func WithMTOM(enabled bool) Option {
```

Comment:
We can drop the bool parameter here safely.

Reply:
Done